pages/customize/map.mdx

```diff
@@ -0,0 +1,122 @@
+collection through an interactive map. When activated, this component displays a map featuring markers representing each
+point Feature within the navPlace properties found throughout the manifests of the overarching collection.
+
+![Canopy Map Example](/docs/map-example.png)
+
+<Section size="2">
+
+  <Heading weight="medium">
+    What are <Strong>navPlace</Strong> Properties
+  </Heading>
+
+  navPlace properties are defined by the <Link href={"https://iiif.io/api/extension/navplace/"}>navPlace extension</Link>.
+
+  A navPlace property may look like this and identifies a single or multiple geographic areas pertinent to a resource
+  using a GeoJSON Feature Collection containing one or more Features.:
+
+  ```json
+  {
+    "navPlace": {
+      "id": "https://digital.lib.utk.edu/notdereferenceable/featurecollection//1",
+      "type": "FeatureCollection",
+      "features": [
+        {
+          "id": "https://digital.lib.utk.edu/notdereferenceable/feature/1",
+          "type": "Feature",
+          "properties": {
+            "label": {
+              "en": [
+                "Interview with Seemona and Daniel Whaley, 2019-09-20 -- Discussion of Gatlinburg, TN"
+              ]
+            }
+          },
+          "geometry": {
+            "type": "Point",
+            "coordinates": [
+              -83.51189,
+              35.71453
+            ]
+          }
+        }
+      ]
+    }
+  }
+  ````
+
+</Section>
+
+<Section size="2">
+
+  <Heading weight="medium">
+    Does Canopy Leverage All <Strong>navPlace</Strong> Properties
+  </Heading>
+
+  <Strong>Not currently</Strong>. Canopy only leverages the `navPlace` properties found in manifests with
+   geometry properties of type `Point`.
+
+</Section>
+
+<Section size="2">
+
+  <Heading weight="medium">
+    Interacting with the Map
+  </Heading>
+
+  From the map component, clicking on a marker will display the label of the Feature and the thumbnail of its parent
+  manifest.
+
+  ![map markers](/docs/clicked-marker.png)
+
+  Clicking the thumbnail or the label will navigate the user to the corresponding work page in Canopy.
+
+</Section>
+
+<Section size="2">
+
+  <Heading weight="medium">
+    Customizing the Map
+  </Heading>
+
+  The Map component can be enabled within `options.json` by switching the value of `enabled` from `false` to `true`.
+
+  ```json
+    {
+      "map": {
+        "defaultBounds": [[51.505, -0.09]],
+        "enabled": true,
+        "icon": {
+          "iconUrl": "images/marker-icon.png",
+          "iconSize": [24, 36],
+          "iconAnchor": [12, 36]
+        },
+        "tileLayers": [
+          {
+            "name": "OpenStreetMap",
+            "url": "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+            "attribution": "&copy; <a href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors"
+          }
+        ]
+      }
+    }
+    ```
```

Comment:
I think this section should be elevated on the page to be inline with customization options for all of these customize/sections. We can also add the copy and filename arguments to the code insert to highlight file location.

```suggestion
  ```json copy filename="config/options.json"
    {
      "map": {
        "defaultBounds": [[51.505, -0.09]],
        "enabled": true,
        "icon": {
          "iconUrl": "images/marker-icon.png",
          "iconSize": [24, 36],
          "iconAnchor": [12, 36]
        },
        "tileLayers": [
          {
            "name": "OpenStreetMap",
            "url": "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
            "attribution": "&copy; <a href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors"
          }
        ]
      }
    }
    ```
```